server/setup.py

```diff
@@ -28,5 +28,16 @@
     install_requires=[
         'blinker', 'celery >=3.1.0, <3.2.0', DJANGO_REQUIRES, 'httplib2', 'iniparse',
         'isodate>=0.5.0', 'm2crypto', 'mongoengine>=0.10.0', 'oauth2>=1.5.211', 'pymongo>=3.0.0',
-        'semantic_version>=2.2.0', 'setuptools']
+        'semantic_version>=2.2.0', 'setuptools'],
+    data_files=[
```

Comment:
Silly question, but...does this actually work? I seem to recall needing some MANIFEST.in shenanigans for this to actually work. Running a `setup.py sdist` makes a source tarball that doesn't contain any html files. Is that expected?